src/main/java/ch/njol/skript/effects/EffTitle.java

```diff
@@ -0,0 +1,91 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+
+@Name("Title")
+@Description("Sends a title/subtitle to the given player(s). Fadein/Fadeout/Stay Times are in ticks")
+@Examples("send title \"Hello player!\" with subtitle \"Welcome to our server\" to player with fadein 10 for 70 with fadeout 20")
+@Since("INSERT VERSION")
+public class EffTitle extends Effect {
+	
+	static {
+		Skript.registerEffect(EffTitle.class, "send [the] title %string% [with subtitle %-string%] to %players% [with fadein %-integer%] [for %-integer%] [with fadeout %-integer%]");
+	}
+	
+	@SuppressWarnings("null")
+	private Expression<String> title;
```

Comment:
For example
```
@SuppressWarnings("null")
private Expression<String> title, subtitle;
@SuppressWarnings("null")
private Expression<Timespan> fadein, stay, fadeout;
```
That way it's not so clustered.

Reply:
Ah... THANK YOU! I was wondering about that, because I too felt it was too cluttered. Done, thank you for the suggestion!